org.eclipse.xtext.xtext.ui.examples/projects/homeautomation/org.eclipse.xtext.example.homeautomation.ui.tests/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,27 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: Xtext Homeautomation Example - UI Tests
+Bundle-Vendor: Eclipse Xtext
+Bundle-Version: 2.15.0.qualifier
+Bundle-SymbolicName: org.eclipse.xtext.example.homeautomation.ui.tests;singleton:=true
+Bundle-ActivationPolicy: lazy
+Require-Bundle: org.eclipse.xtext.example.homeautomation.ui,
+ org.eclipse.xtext.testing,
+ org.eclipse.core.runtime,
+ org.eclipse.ui.workbench;resolution:=optional,
+ org.eclipse.xtext.common.types.ui,
+ org.eclipse.jdt.core,
+ org.eclipse.xtext.xbase.testing,
+ org.eclipse.xtext.ui.testing,
+ org.eclipse.xtext.junit4,
```

Comment:
do we still need this or is xtext.testing supposed to be sufficient?

Reply:
Thanks for the hint! I will investigate that and (wherever possible) clean up the Require-Bundle sections of all the examples in a follow up PR.